ui/src/app/icons/notebook-icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+export const NotebookIcon = ({style = {}}) =>
+  <svg style={{height: '45px', width: '36px', ...style}}
```

Comment:
Was this path hand-coded into this component, or was this code generated from something else? If the latter, then we should make sure to stash the input(s) somewhere.

Reply:
Sorry, I don't know where this comes from.  This existed in the Angular version.